templates/object-template.conf.j2

```diff
@@ -1,7 +1,7 @@
 // {{ ansible_managed }}
 {% import 'item_definitions.j2' as definitions %}
 {% if hostvars[item].i2_objects is defined -%}
-{% for item in hostvars[item].i2_objects -%}
+{% for item in hostvars[item].i2_objects|dict2items -%}
```

Comment:
Thanks, I was trying to debug that, but jinja/ansible's error messages weren't really helpful.